### PR TITLE
ACP2E-342: [Documentation] Mention that partially refunded order remains in Processing status until all items are being shipped

### PR DESCRIPTION
**Pull request by vkolesny**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) is to mention in the Order Status documentation that partially refunded order remains in Processing status until all ordered items (including refunded items) are being shipped.
This means that order status will not be changed to Complete, while at least 1 order item is not shipped.
This behavior was confirmed by the Product Owner in the following JIRA comment:
https://jira.corp.magento.com/browse/MAGETWO-95724?focusedCommentId=1055519&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1055519

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

Magento 2.4.x

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

Both Magento Open Source and Adobe Commerce

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

No

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/sales/order-status.html

### CODE REVIEW

File: src/sales/order-status.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ All orders have an order status that is associated with a stage in the order pro
 ![]({% link sales/assets/stores-order-status-column.png %}){: .zoom}
 _Order Status_
 
+{:.bs-callout-tip}
+Partially refunded order remains in `Processing` status until **_all_** ordered items (including refunded items) are being shipped. Such order status will not be changed to `Complete`, while at least 1 order item is not shipped.
```
Comment (leslietilling) [inline src/sales/order-status.md:13]:
```suggestion
A partially refunded order remains in `Processing` status until **_all_** ordered items (including refunded items) are shipped. The order status does not change to `Complete` when even one order item is not yet shipped.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some tweaking for clarity and language

File: src/sales/order-status.md (hunk 1 above)
Comment (vkolesny) [reply src/sales/order-status.md]: Suggestions successfully committed.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**